floss/language/go/extract.py

```diff
@@ -0,0 +1,431 @@
+        .text:0000000000426BCF BB 18 00 00 00                mov     ebx, 18h
+        .text:0000000000426BD4 E8 67 CB 00 00                call    runtime_throw
+        """
+        longstring64 = re.compile(b"\x48\x8d(?=.(....).(....))", re.DOTALL)
+
+        """
+        .text:000000000048E780 	48 83 FB 13 	cmp rbx, 13h
+        .text:000000000048E784 	75 13 	jnz short loc_48E799
+        .text:000000000048E786 	48 89 D9 	mov rcx, rbx
+        .text:000000000048E789 	48 8D 1D E1 B5 01 00 	lea rbx, unk_4A9D71
+        """
+        longstring64_2 = re.compile(b"\x48\x83(?=.(.)(.){2,5}\x48\x8D.(....))", re.DOTALL)
+
+        """
+        .text:0000000000481745 48 C7 40 08 17 00 00 00       mov     qword ptr [rax+8], 17h
+        .text:000000000048174D 48 8D 0D A2 AC 02 00          lea     rcx, aSyntaxErrorInP ; "syntax error in pattern"
+        .text:0000000000481754 48 89 08                      mov     [rax], rcx
+        """
+        longstring64_3 = re.compile(b"\x48\xc7(?=..(.)...\x48\x8D.(....))", re.DOTALL)
+
+        """
+        .text:00000000004033CD B9 1C 00 00 00                mov     ecx, 1Ch
+        .text:00000000004033D2 48 89 C7                      mov     rdi, rax
+        .text:00000000004033D5 48 89 DE                      mov     rsi, rbx
+        .text:00000000004033D8 31 C0                         xor     eax, eax
+        .text:00000000004033DA 48 8D 1D C3 A1 0A 00          lea     rbx, aComparingUncom ; "comparing uncomparable type "
+        .text:00000000004033E1 E8 5A 63 04 00                call    runtime_concatstring2
+        """
+        longstring64_4 = re.compile(b"\xb9(?=(.)...........\x48\x8D.(....))", re.DOTALL)
+
+    elif pe.FILE_HEADER.Machine == pefile.MACHINE_TYPE["IMAGE_FILE_MACHINE_I386"]:
+        """
+        .data:102A78D0 E3 9A 17 10                       dd offset aString
+        .data:102A78D4 12                                db  12h
+        """
+        alignment = 0x8
+        fmt = "<II"
+
+        """
+        .text:0048CED3 75 6D                                         jnz     short loc_48CF42
+        .text:0048CED5 81 7D 00 50 61 73 73                          cmp     dword ptr [ebp+0], 73736150h
+        .text:0048CEDC 75 64                                         jnz     short loc_48CF42
+        .text:0048CEDE 66 81 7D 04 77 6F                             cmp     word ptr [ebp+4], 6F77h
+        .text:0048CEE4 75 5C                                         jnz     short loc_48CF42
+        .text:0048CEE6 80 7D 06 72                                   cmp     byte ptr [ebp+6], 72h ; 'r'
+        .text:0048CEEA 75 56                                         jnz     short loc_48CF42
+        """
+        combinedregex = re.compile(
+            b"\x81\xf9(....)|\x81\x38(....)|\x81\x7d\x00(....)|\x81\x3B(....)|\x66\x81\xf9(..)|\x66\x81\x7b\x04(..)|\x66\x81\x78\x04(..)|\x66\x81\x7d\x04(..)|\x80\x7b\x06(.)|\x80\x7d\x06(.)|\x80\xf8(.)|\x80\x78\x06(.)",
```

Comment:
I'm curious how common this pattern actually is, seems rather specific to the test sample, but we can see how often it hits later on